### PR TITLE
small tweaks to Readme

### PR DESCRIPTION
**Pull request by akiezun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,21 +30,23 @@ General guidelines for Hellbender developers
 
 * Hellbender is  BSD licensed.  The license is in the top level LICENSE.TXT file.  Do not add any additional license text or accept files with a license included in them.
 
-* Untested code is considered **non-existent** and thus is subject to removal at any time (exception is main methods, or super corner conditions, or `toString()` code). New code without tests should not be accepted as pull requests.
+* Untested code is considered **non-existent** and thus is subject to removal at any time (exception is main methods, or super corner conditions, or `toString()` code).
 
-* Do use Java 8 features liberally, in particular streams, functional programming etc.
+* Don't accept pull requests that introduce warnings. Warnings need to be addressed or supressed.
```
Comment (lbergelson) [inline README.md]: We should add this check to travis if we want to enforce it.  Adding a ticket for this #368.

**Comment by lbergelson** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ To run all tests, run `gradle test`. Report is in `build/reports/tests/index.htm
 
 To run a single test class, run something like this, `gradle test -Dtest.single=ReadUtilsUnitTest`.
 
-To run tests and compute coverage reports, run `gradle jacocoTestReport`. The report is then in `build/reports/jacoco/test/html/index.html`.
+To run tests and compute coverage reports, run `gradle jacocoTestReport`. The report is then in `build/reports/jacoco/test/html/index.html`. (IntelliJ 14 has a good coverage tool that is prefereable for development).
```
Comment (droazen) [inline README.md]: prefereable -> preferable

File: README.md (hunk 1 above)
Comment (droazen) [inline README.md]: supressed -> suppressed

File: README.md [hunk 3]
```diff
@@ -30,21 +30,23 @@ General guidelines for Hellbender developers
...
+* Untested code is considered **non-existent** and thus is subject to removal at any time (exception is main methods, or super corner conditions, or `toString()` code).
 
-* Do use Java 8 features liberally, in particular streams, functional programming etc.
+* Don't accept pull requests that introduce warnings. Warnings need to be addressed or supressed.
 
-* Don't override `clone()` unless you really know what you're doing. If you do override it, document thoroughly.
+* Don't accept pull requests that significantly decrease coverage (less than 1% decrease is sort of tolerable). 
 
-* Don't override or call `finalize()` even when you think you know what you're doing.
+* All newly code must have good test coverage (>90%).
```
Comment (droazen) [inline README.md]: newly code -> newly-written code

Comment (lbergelson) [inline README.md]: I read right over this without seeing it.

**Comment by lbergelson** [comment]
Is there  some sort of spellcheck for github?

File: README.md (hunk 3 above)
Comment (droazen) [inline README.md]: This is why we need reviews even for things like README changes :)

**Comment by akiezun** [comment]
back to @lbergelson for review

**Comment by lbergelson** [comment]
squash and then :+1:
